libs/components/forms/src/lib/modules/file-attachment/file-drop/file-drop.component.ts

```diff
@@ -254,22 +263,60 @@ export class SkyFileDropComponent implements OnDestroy {
 
   #_minFileSize = MIN_FILE_SIZE_DEFAULT;
 
+  #notifyTouched: (() => void) | undefined;
+  #notifyChange:
+    | ((_: (SkyFileItem | SkyFileLink)[] | undefined | null) => void)
+    | undefined;
+  #_uploadedFiles: (SkyFileItem | SkyFileLink)[] | undefined | null = [];
+
   readonly #fileAttachmentService = inject(SkyFileAttachmentService);
   readonly #liveAnnouncerSvc = inject(SkyLiveAnnouncerService);
   readonly #resourcesSvc = inject(SkyLibResourcesService);
   readonly #idSvc = inject(SkyIdService);
 
   protected errorId = this.#idSvc.generateId();
+
+  protected ngControl = inject(NgControl, { optional: true });
+
   protected rejectedFiles: SkyFileItem[] = [];
 
+  constructor() {
+    if (this.ngControl) {
+      this.ngControl.valueAccessor = this;
+    }
+  }
+
   public ngOnDestroy(): void {
     this.filesChanged.complete();
     this.linkChanged.complete();
     this.linkInputBlur.complete();
   }
 
+  public writeValue(value: any): void {
```

Comment:
Can the `value` be something besides `any`? Can you create a known type so it's easier to adopt for our consumers?

Reply:
if i set it to anything other than `any`, the `setValue` function on the formControl doesnt ever use the `writeValue` function

Reply:
I'm not following. If you set this to `unknown`, does it work then?